File: Leetcode/sbgo/Leetcode1716-计算力扣银行的钱.go
package sbgo

func totalMoney(n int) int {
	ans, cnt := 0, 1
	for i := 0; i < n; i++ {
		if i != 0 && i%7 == 0 {
			cnt++
		}
		ans += (cnt + i%7)
	}
	return ans
}

File: Leetcode/Rust/Leetcode1716-计算力扣银行的钱.rs
impl Solution {
    pub fn total_money(n: i32) -> i32 {
        let (mut ans,mut cnt) = (0,1);
        for i in 0..n{
            if i != 0 && i % 7 == 0 {
                cnt += 1;
            }
            ans += (cnt + i % 7);
        }
        ans
    }
}